src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -0,0 +1,16 @@
+@import '../../../common/variables';
+
+.bcs-ActivityThread-toggle {
+    margin-left: $sidebarActivityFeedSpacingHorizontal;
+    color: $bdl-box-blue;
+    font-weight: bold;
+
+    &:hover {
```

Comment:
Do we need to add focus styles, as well?

Reply:
focus is fine but `:active` needed to be handled 👍